### ISSUE TITLE
Select Previous Block (Ctrl+UP) doesn't work anymore when the focus is in the input field

### ISSUE THREAD

**Issue by hron** [warpdotdev/Warp#5115]
### Dupe Check

- [X] I have [searched Warp bugs](https://github.com/warpdotdev/warp/issues?q=is%3Aissue+label%3ABugs) and there are no duplicates

### Describe the bug

There is no way to move keyboard focus to the blocks (command outputs) from the command input block anyway. Before `v0.2024.07.09.08.01.stable_00` it was possible to hit Ctrl-Up and the keyboard focus moved from the Command Input to the latest block. Now it doesn't work

### To reproduce

1. Run any command, e.g. pwd
2. Press Ctrl+UP

### Expected behavior

The keyboard focus should be moved from the command input to the latest block (output of `pwd`). Instead nothing happens.

### Screenshots

[Screencast from 2024-07-12 12-04-43.webm](https://github.com/user-attachments/assets/ddd67685-5e6e-471e-a542-43caf67a1fe2)

### Operating system

Linux

### Operating system and version

Ubuntu 24.04

### Shell Version

bash

### Current Warp version

v0.2024.07.09.08.01.stable_00

### Regression

Yes, this bug started recently or with an X Warp version

### Does this block you from using Warp daily?

Yes, this issue prevents me from using Warp daily.

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b9d78064-c89e-4973-b153-5178a31ee54e

None

**Comment by dannyneira** [comment]
Hi @hron thanks for letting us know about this issue. I can reproduce this and will notify the team and we'll post any updates on this thread.

To anyone else facing this issue, please add a 👍 to the original post at the top or comment with your details, and subscribe if you'd like to be notified.

**Comment by shicaiv** [comment]
I have it also on linux opensuse...

**Comment by dannyneira** [comment]
Hi Folks, this issue has been resolved in the latest version of Warp. Closing as such, but please update and let us know if the issue persists.
